native_client/ctcdecode/scorer.cpp

```diff
@@ -69,45 +71,60 @@ void Scorer::setup_char_map()
   }
 }
 
-int Scorer::load_lm(const std::string& lm_path)
+int Scorer::load_lm(const std::string& lm_string, bool load_from_bytes)
```

Comment:
Either provide a default value, split the implementation in two `load_lm` or update all call sites: this PR does not build because of `load_lm` calls in `generate_scorer_package.cpp`

Reply:
I've provided a default value (false) for the boolean